lib/scripts/mtw.js

```diff
@@ -107,6 +103,14 @@ export class ContentScript {
       style.href  = chrome.extension.getURL('/assets/css/MTWStyles.css');
       document.getElementsByTagName('head')[0].appendChild(style);
       document.styleSheets[0].insertRule('span.mtwTranslatedWord {' + cssStyle + '}', 0);
+      /**
```

Comment:
I thought a bug was introduced while merging, will fix this

On Tuesday 21 June 2016, Ankit Muchhala <notifications@github.com> wrote:

> In lib/scripts/mtw.js
> <https://github.com/OiWorld/MindTheWord/pull/141#discussion_r67844736>:
>
> > @@ -107,6 +103,14 @@ export class ContentScript {
> >        style.href  = chrome.extension.getURL('/assets/css/MTWStyles.css');
> >        document.getElementsByTagName('head')[0].appendChild(style);
> >        document.styleSheets[0].insertRule('span.mtwTranslatedWord {' + cssStyle + '}', 0);
> > +      /**
>
> @rohankatyal29 <https://github.com/rohankatyal29> why are you adding them
> here? Look into MTWStyles.css. They are already there. Also if you want
> to add any CSS styles, add directly in that file.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/OiWorld/MindTheWord/pull/141/files/0ae51855e88846b5b43d7dc50aa547b733e0908c#r67844736>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ACVryg2mtpJpqo-HHZPl3P8MM8wx63Kyks5qN733gaJpZM4I6lLq>
> .
>


-- 
Regards,
Rohan Katyal

Please ignore the typos and brevity, email sent using a small form factor
device